app/ticket/components/CheckOut.tsx

```diff
@@ -0,0 +1,158 @@
+import { formatAmount } from '@/utils/formatAmount';
+import { ArrowLeft, ArrowUpRight } from 'lucide-react';
+import { useRouter } from 'next/navigation';
+import React from 'react';
+
+const TicketPackages = [
+  {
+    key: 'dayOne',
+    day: 'Friday',
+    activityType: 'Workshop',
+    price: 4_000,
+  },
+  {
+    key: 'dayTwo',
+    day: 'Saturday',
+    activityType: 'Main event & Workshop',
+    price: 4_000,
+  },
+  {
+    key: 'bothDays',
+    day: 'Friday & Saturday',
+    activityType: 'Main event & Workshop',
+    price: 8_000,
+  },
+];
+
+interface CheckOutProps {
+  selectedPackage: {
+    key: string;
+    day: string;
+    activityType: string;
+    price: number;
+  };
+  setSelectedPackage: React.Dispatch<
+    React.SetStateAction<{
+      key: string;
+      day: string;
+      activityType: string;
+      price: number;
+    }>
+  >;
+  formData: {
+    fullName: string;
+    email: string;
+  };
+  setFormData: React.Dispatch<
+    React.SetStateAction<{
+      fullName: string;
+      email: string;
+    }>
+  >;
+  setView: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const CheckOut: React.FC<CheckOutProps> = ({
+  selectedPackage,
+  setSelectedPackage,
+  formData,
+  setFormData,
+  setView,
+}) => {
+  const router = useRouter();
+  return (
+    <form
+      className="bg-white px-4 md:px-28 py-28 rounded-2xl flex flex-col gap-5 border md:border-none"
+      onSubmit={(e) => {
+        e.preventDefault();
+        setView('summary');
+      }}
+    >
+      <button
+        className="w-fit flex flex-col md:flex-row justify-center items-start mdl:items-center gap-3 text-gray-400"
+        onClick={() => router.back()}
+      >
+        <ArrowLeft />
+        <p className="font-bold text-xl md:text-2xl">Check Out</p>
```

Comment:
The color of this text doesn't tally with that of the design.

Design
<img width="817" height="165" alt="image" src="https://github.com/user-attachments/assets/7f8628a4-5e3b-409e-8a0b-4d600493b2c1" />

Your implementation
<img width="764" height="103" alt="image" src="https://github.com/user-attachments/assets/a9514c21-3157-4328-bb3f-778b09b42c6d" />